### ISSUE TITLE
Editor fails to render content if table syntax is wrong

### ISSUE THREAD

**Issue by kkstoklund** [nhn/tui.editor#2050]
## Describe the bug
I'm working on implementing the editor in an internal document management system.
When editing an existing document, the markdown content is fetched from the DB and inserted into the editor upon initiation using the setMarkdown() method.
I've encountered an issue where the editor simply breaks when calling setMarkdown() if the content contains a table that has more table headers than delimiters - see example: 

![Screen Shot](https://user-images.githubusercontent.com/29021651/143032827-e41b6684-6318-4042-83c4-0b543cd0979c.png)
_3 headers with only 2 delimiters_

![Screen Shot 2021-11-23 at 14 33 27](https://user-images.githubusercontent.com/29021651/143035609-beef81c0-710e-4e92-9456-8cd6ffcd6f65.png)
_Chrome console output_

## Expected behavior
I would expect the editor to handle this more gracefully, perhaps by rendering the content regardless (which works if you paste above markdown example into a new document, where content does not need to be set on pageload).

## Desktop (please complete the following information):
 - OS: macOS, Windows 10
 - Browser Chrome, Firefox, Safari
 - Version 3.1.1

## Additional context
````javascript
var editor = new toastuiEditor({
  el: document.querySelector('#docToastEditor'),
  minHeight: '500px',
  height: 'auto',
  initialEditType: 'wysiwyg',
  previewStyle: 'vertical',
  previewHighlight: true,
  hooks: {
      addImageBlobHook: function() {return false}
  },
  toolbarItems: [
      ['heading', 'bold', 'italic', 'strike'],
      ['hr', 'quote'],
      ['ul', 'ol', 'indent', 'outdent'],
      ['table', 'link',
          {
              el: uploadBtn(),
              tooltip: 'Upload file'
          }
      ],
      ['code', 'codeblock'],
      [
          {
              el: linkToGuideBtn(),
              tooltip: 'Markdown guide'
          }
      ]
  ]
});

// textarea element when editing existing document
let content = document.getElementById('documentation_content_content');

editor.setMarkdown(content.value);
````

**Comment by js87zz** [comment]
@kkstoklund 
I checked the issue. It will be improved.

**Comment by kkstoklund** [comment]
Actually, the issue can be replicated we easily from here [https://ui.toast.com/tui-editor](https://ui.toast.com/tui-editor)

While in markdown mode:

1. Paste this into the editor
`| col1 | col2 | col3 |
| --- | --- |
| cell1 | cel2 | cel3 |`
2. Switch to WYSIWYG mode.
3. Check the web console output.

**Comment by js87zz** [comment]
@kkstoklund 
The bug has been fixed in `v3.1.2`. Thanks!
